components/nexus-core/src/main/java/org/sonatype/nexus/internal/orient/LogConfigurationCustomizerImpl.java

```diff
@@ -31,7 +31,7 @@
 {
   @Override
   public void customize(final Configuration config) {
-    config.setLoggerLevel("org.sonatype.nexus.orient", DEFAULT);
+    config.setLoggerLevel("org.sonatype.nexus.test.orient", DEFAULT);
```

Comment:
side-effect of the refactoring... this should be left as "org.sonatype.nexus.orient" as this is non-test code